comet/utils/redis.py

```
@@ -0,0 +1,215 @@
+                return json.loads(value)
+            except json.JSONDecodeError:
+                return value
+                
+        except (ConnectionError, TimeoutError) as e:
+            logger.warning(f"Redis connection issue during GET for key {self._sanitize_key_for_logging(key)}: {e}")
+            self._connected = False
+            return None
+        except RedisError as e:
+            logger.error(f"Redis error during GET for key {self._sanitize_key_for_logging(key)}: {e}")
+            return None
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)